controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/application/ApplicationApiHandler.java

```diff
@@ -232,7 +232,7 @@ public HttpResponse auditAndHandle(HttpRequest request) {
         }
         catch (RuntimeException e) {
             log.log(Level.WARNING, "Unexpected error handling '" + request.getUri() + "'", e);
-            return ErrorResponse.internalServerError(Exceptions.toMessageString(e));
+            return ErrorResponse.internalServerError("Unexpected error occurred");
```

Comment:
Should we perhaps generate a UUID here, and place with both the response, and in the log, to make it easier to find relevant log entries when a user asks what happened?

Reply:
... or is that unnecessary?